dashboard/config/locales/ru-RU.yml

```diff
@@ -200,7 +200,7 @@ ru:
     student_terms_markdown: Адреса электронной почты не используются для контакта
       с учениками. Ученики никогда не будут получать электронные письма от Code.org,
       за исключением восстановления пароля. Просмотрите наши [правила по политике
-      конфиденциальности](%{privacy_policy_url}) для дальнейшей информации.
```

Comment:
Just want to confirm - is this change the first for the 5XX errors on sign up in Russian or a re-introduction of the problem?

Reply:
Checked https://github.com/code-dot-org/code-dot-org/pull/32272/files This is the fix.

 Is there any reason to keep the 2 hot-fix back merges separate?